sql-db/sql-app/src/test/resources/test.properties

```diff
@@ -1,6 +1,3 @@
 ts.app.log.enable=true
 ts.database.log.enable=true
-ts.database.openshift.use-internal-service-as-url=true
-ts.mssql.log.enable=true
-ts.mssql.openshift.template=/mssql.yaml
```

Comment:
Is dropping this custom template fixing anything, or has it just become obsolete?

Reply:
Yes, it's fixing the OpenShift Mssql scenario. This scenario without this configuration works fine.

Reply:
I see. I didn't notice the scenario failing. Thanks for the explanation.